src/konnektor/network_planners/concatenators/mst_concatenator.py

```diff
@@ -49,15 +49,15 @@ def __init__(
             scorer=scorer,
             network_generator=MstNetworkAlgorithm(),
             n_processes=n_processes,
-            _initial_edge_lister=None,
+            _initial_edge_lister=None,  ## TODO: should this be _initial_edge_lister?
```

Comment:
I'm not sure if i understand this comment correctly, as before and after looks the same?

Reply:
It looks like that in the parent class, the `_initial_edge_lister=None`, however, in the doc string it says 
```
_initial_edge_lister: LigandNetworkPlanner, optional
            this LigandNetworkPlanner is used to give the initial set of edges.
             For standard usage, the Maximal NetworPlanner is used.
            However in large scale approaches, it might be interesting to use
             the heuristicMaximalNetworkPlanner. (default: None)
```
In this class `MSTConcatenator` it is listed as `_initial_edge_lister: NetworkConcatenator = None,`, while I think it should be a `LigandNetworkPlanner`? Or are those two connected?

In lines 85-89, all possible edges between the networks are created, therefore I would think that the `initial_edge_lister` is not needed, so None could be ok?!

Reply:
it is valid for it to be `None`, but the issue here is that if `_initiali_edge_lister` is defined in the `MstConcatenator` class, it never gets passed to the `super()__init__()` call.

Reply:
I agree with @atravitz - if `_initial_edge_lister` is part of the signature it should be passed through instead of hard-coding to ``None``. The argument should also be listed in the class docstring.

Furthermore, we should avoid "private" kwargs in a user-facing class method (i.e. anything starting with a "_" that's bad form imho).

Fixing in a separate issue/PR combo is fine, especially since it seems like we'll need to rename the variable.

Reply:
issue opened [here](https://github.com/OpenFreeEnergy/konnektor/issues/99)